### ISSUE TITLE
build: no native function or rule 'http_archive'

### ISSUE THREAD

**Issue by sergey-safarov** [hiveco/conntrack_exporter#13]
on the master

```
bazel build -c dbg //:conntrack_exporter
DEBUG: Rule 'prometheus_cpp' indicated that a canonical reproducible form can be obtained by modifying arguments shallow_since = "1509747915 -0700"
DEBUG: Repository prometheus_cpp instantiated at:
  /tmp/conntrack_exporter/WORKSPACE:3:15: in <toplevel>
Repository rule git_repository defined at:
  /root/.cache/bazel/_bazel_root/d91834a1ac54014cf5ba7f51135e5a3c/external/bazel_tools/tools/build_defs/repo/git.bzl:199:33: in <toplevel>
ERROR: Traceback (most recent call last):
	File "/tmp/conntrack_exporter/WORKSPACE", line 10, column 28, in <toplevel>
		prometheus_cpp_repositories()
	File "/root/.cache/bazel/_bazel_root/d91834a1ac54014cf5ba7f51135e5a3c/external/prometheus_cpp/repositories.bzl", line 129, column 29, in prometheus_cpp_repositories
		load_com_google_protobuf()
	File "/root/.cache/bazel/_bazel_root/d91834a1ac54014cf5ba7f51135e5a3c/external/prometheus_cpp/repositories.bzl", line 99, column 11, in load_com_google_protobuf
		native.http_archive(
Error: no native function or rule 'http_archive'
Available attributes: aar_import, action_listener, alias, android_binary, android_device, android_device_script_fixture, android_host_service_fixture, android_instrumentation_test, android_library, android_local_test, android_ndk_repository, android_sdk, android_sdk_repository, android_tools_defaults_jar, apple_cc_toolchain, available_xcodes, bazel_version, bind, cc_binary, cc_host_toolchain_alias, cc_import, cc_libc_top_alias, cc_library, cc_proto_library, cc_shared_library, cc_shared_library_permissions, cc_test, cc_toolchain, cc_toolchain_alias, cc_toolchain_suite, config_feature_flag, config_setting, constraint_setting, constraint_value, environment, existing_rule, existing_rules, exports_files, extra_action, fdo_prefetch_hints, fdo_profile, filegroup, genquery, genrule, glob, j2objc_library, java_binary, java_import, java_library, java_lite_proto_library, java_package_configuration, java_plugin, java_plugins_flag_alias, java_proto_library, java_runtime, java_test, java_toolchain, label_flag, label_setting, local_config_platform, local_repository, new_local_repository, objc_import, objc_library, package_group, package_name, platform, propeller_optimize, proto_lang_toolchain, proto_library, py_binary, py_library, py_runtime, py_test, register_execution_platforms, register_toolchains, repository_name, sh_binary, sh_library, sh_test, subpackages, test_suite, toolchain, toolchain_type, xcode_config, xcode_config_alias, xcode_version
ERROR: Traceback (most recent call last):
	File "/tmp/conntrack_exporter/WORKSPACE", line 13, column 32, in <toplevel>
		conntrack_exporter_dependencies()
	File "/tmp/conntrack_exporter/repositories.bzl", line 40, column 24, in conntrack_exporter_dependencies
		argagg_repositories()
	File "/tmp/conntrack_exporter/repositories.bzl", line 30, column 11, in argagg_repositories
		native.new_git_repository(
Error: no native function or rule 'new_git_repository'
Available attributes: aar_import, action_listener, alias, android_binary, android_device, android_device_script_fixture, android_host_service_fixture, android_instrumentation_test, android_library, android_local_test, android_ndk_repository, android_sdk, android_sdk_repository, android_tools_defaults_jar, apple_cc_toolchain, available_xcodes, bazel_version, bind, cc_binary, cc_host_toolchain_alias, cc_import, cc_libc_top_alias, cc_library, cc_proto_library, cc_shared_library, cc_shared_library_permissions, cc_test, cc_toolchain, cc_toolchain_alias, cc_toolchain_suite, config_feature_flag, config_setting, constraint_setting, constraint_value, environment, existing_rule, existing_rules, exports_files, extra_action, fdo_prefetch_hints, fdo_profile, filegroup, genquery, genrule, glob, j2objc_library, java_binary, java_import, java_library, java_lite_proto_library, java_package_configuration, java_plugin, java_plugins_flag_alias, java_proto_library, java_runtime, java_test, java_toolchain, label_flag, label_setting, local_config_platform, local_repository, new_local_repository, objc_import, objc_library, package_group, package_name, platform, propeller_optimize, proto_lang_toolchain, proto_library, py_binary, py_library, py_runtime, py_test, register_execution_platforms, register_toolchains, repository_name, sh_binary, sh_library, sh_test, subpackages, test_suite, toolchain, toolchain_type, xcode_config, xcode_config_alias, xcode_version
ERROR: error loading package 'external': Package 'external' contains errors
INFO: Elapsed time: 0.087s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
make: *** [Makefile:4: build] Error 1
```

**Comment by dbendelman** [comment]
@sergey-safarov I’m not actively developing this but happy to merge a fix if you figure it out.

**Comment by dbendelman** [comment]
Fixed in #14
